app/src/main/java/com/duckduckgo/app/browser/WebDataManager.kt

```diff
@@ -83,16 +83,10 @@ class WebViewDataManager @Inject constructor(
         fileDeleter.deleteContents(File(dataDir, WEBVIEW_DEFAULT_DIRECTORY_NAME), exclusions)
     }
 
-    /**
-     * Deprecated and not needed on Oreo or later
-     */
-    @Suppress("DEPRECATION")
-    private fun clearFormData(webViewDatabase: WebViewDatabase) {
-        webViewDatabase.clearFormData()
-    }
-
-    private fun clearAuthentication(webViewDatabase: WebViewDatabase) {
-        webViewDatabase.clearHttpAuthUsernamePassword()
+    private fun clearAuthentication(webView: WebView) {
+        GlobalScope.launch(Dispatchers.IO) {
```

Comment:
why is this triggered as a new global coroutine? That will return without ensuring it was completed.

Reply:
Yes, I will follow the same approach I did for other methods and remove the `GlobalScope` here